ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/patch-panel.j2

```diff
@@ -9,6 +9,20 @@
 ## Patch Panel
 
 ### Patch Panel Summary
+{%     if patch_panel.connector.interface.recovery.review_delay.min is arista.avd.defined %}
+
+Patch Panel Connector Interface Recovery Review Delay Min: {{ patch_panel.connector.interface.recovery.review_delay.min }}
+{%     endif %}
+{%     if patch_panel.connector.interface.recovery.review_delay.max is arista.avd.defined %}
+
+Patch Panel Connector Interface Recovery Review Delay Max: {{ patch_panel.connector.interface.recovery.review_delay.max }}
+{%     endif %}
```

Comment:
here the logic is slightly different from the config template in the sense that if I only give min delay it wont' show up in the config but will show up in the doc (as the if statements are disjoint for doc)

Reply:
hmm ok with the schema (assuming it is not disabled) it won't happen so let's ignore this.

Reply:
Hi Guillaume,  i could also check if min AND max is set and only if this is the case render both outputs on the doc.. does that make more sense?